css-two-column-layout/styles.css

```diff
@@ -0,0 +1,63 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  flex-wrap: nowrap;
+  justify-content: center;
```

Comment:
`flex-wrap` & `justify-content` will only have an impact if they are used alongside a `display: flex`. Since there's no `display: flex` here, these aren't doing anything.